src/components/button/__docs__/Button.mdx

```diff
@@ -15,26 +14,32 @@ Button component with different props.
 ## Usage
 
 ```ts
-import {Button} from "sld-ui";
+import { Button } from '@gloabal-regulatory-writing-consulting/gxt-components';;
+
+const ExampleButton = () => {
+ const handleClick = (event: React.MouseEvent) => {
+    // Handle button click
+  };
 
-const Example = () => {
   return (
-      <Button
-        size={"small"}
-        text={"Button"}
-        onClick={()=> console.log("Clicked")}
-        primary
-      />
+    <Button
+      title="Button"
+      onClick={handleClick}
+      disabled={false}
+      size="medium"
+      variant="primary"
+    />
   );
 };
 
-export default Example;
+export default ExampleButton;
 ```
 
 #### Arguments
 
-- **text** _`() => void`_ - A string that represents the text content of the button.
-- **primary** - A boolean indicating whether the button should have a primary styling or not. Typically, a primary button stands out as the main action in a user interface.
-- **disabled** - A boolean indicating whether the button should be disabled or not. When disabled, the button cannot be clicked or interacted with.
-- **size** - A string with one of three possible values: "small," "medium," or "large." It defines the size or dimensions of the button.
-- **onClick** - A function that is called when the button is clicked. It receives a MouseEventHandler for handling the click event on the button element.
\ No newline at end of file
+- **title** - Text or node to render inside the button.
+- **loading** -  A boolean indicating whether to show a loading spinner in the button.
+- **disabled** -  A boolean indicating whether the button should be disabled.
+- **classNames** - Additional CSS classes to apply to the button.
```

Comment:
Let's also get rid of this, as we can make those changes within the app in which it is being consumed.